designer/server/src/main/scala/pl/touk/nussknacker/ui/util/GenerateDesignerOpenApiYamlFile.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.ui.util
+
+import better.files.Dsl._
+import cats.effect.{ExitCode, IO, IOApp}
+import com.typesafe.scalalogging.StrictLogging
+import pl.touk.nussknacker.ui.services.NuDesignerApiAvailableToExpose
+
+object GenerateDesignerOpenApiYamlFile extends IOApp with StrictLogging {
+
+  override def run(args: List[String]): IO[ExitCode] = for {
+    _ <- IO.delay(logger.info("Generating Nu Designer OpenAPI document ..."))
+    _ <- generateAndSave()
+    _ <- IO.delay(logger.info("DONE!"))
+  } yield ExitCode.Success
+
+  private def generateAndSave() = IO {
+    (pwd / "docs" / "api" / "internal" / "nu-designer-openapi.yaml")
```

Comment:
The docs directory is treated specially - I think all files there are used in building public docs. Do we want to integrate this into our public docs? Our pipeline also labels changes that modify these files and treats them differently. Have you considered this?

Reply:
nope. Didn't know about it. I'm not sure if we should treat it as external docs currently. Probably not. 
WDYT about moving it to some `internal-docs` folder instead? Because it's kind of docs.

Reply:
Yes, I think moving it to some other folder is the best solution for now. Maybe use resources folder as arkadius suggested?

Reply:
it's not a resource. It's a doc. So, I'll go with the new folder idea

Reply:
It is docs for developers, so don't worry - we will find it no matter where you put it ;) But `internal-docs` is fine for me. Only, please use the resources mechanism to determine the path of this file in tests. You can find an example e.g. in `BaseNuRuntimeBinTestMixin:32`

Reply:
because it's not a resource I think I cannot use Java's Resource. But I can use the `user.dir` system property. WDYT about this approach?

Reply:
I got your intention now. It's fixed